script.js

```diff
@@ -0,0 +1,12 @@
+
+<script>
```

Comment:
These script tags are HTML. When we have JavaScript in its own file we don't need them. In fact it throws an error.